### PR TITLE
more efficient ifs

### PR DESCRIPTION
**Pull request by Euro20179**
add-playlist got changed to playlist-add because it doesn't add a playlist, it adds to a playlist (also in help it displayed playlist-add)

### CODE REVIEW

File: ytfzf [hunk 1]
```diff
@@ -818,7 +818,7 @@ get_search_query () {
 			printf "$search_prompt"
 			read -r search_query
 		fi
-		[ -z "$search_query" ] && clean_up && exit 0
+		[ -z "$search_query" ] && (clean_up; exit 0)
```
Comment (pystardust) [inline ytfzf:821]: I don't know if this is intensional but `()` will run the code in a sub shell (won;t make much of a difference here). Use `{}` for grouping commands with out subshell

Comment (Euro20179) [reply ytfzf:821]: () makes exit only happen if [ -z $search_query Is true

Comment (pystardust) [reply ytfzf:821]: if you do `(exit 0)` it will not exit the script. It will only exit the subshell

Comment (Euro20179) [reply ytfzf]: fixed

**Pull request merged**